python/looker_sdk/rtl/transport.py

```diff
@@ -63,7 +64,8 @@ class PTransportSettings(Protocol):
     headers: Optional[MutableMapping[str, str]]
 
     def is_configured(self) -> bool:
-        return bool(self.base_url)
+        if not self.base_url:
+            raise error.SDKError("Missing required configuration value: base_url")
```

Comment:
I've gone back and forth on this but my latest thinking is that raising here rather than everywhere else is cleaner